src/charm_state.py

```diff
@@ -202,3 +223,18 @@ def flask_port(self) -> int:
             The port number to use for the Flask server.
         """
         return 8000
+
+
+KNOWN_CHARM_CONFIG = (
+    "webserver_workers",
+    "webserver_threads",
+    "webserver_keepalive",
+    "webserver_timeout",
```

Comment:
I don't think we need to define these parameters here as they can't get in conflict

Reply:
This list is just used for filtering out user-defined configurations, the conflict handling happens in `FlaskApp.flask_environment`. The reason I didn't perform the merging and conflict removing earlier is to preserve as much information as possible for charm to decide what to do.

Reply:
What I meant is that a user won't be able to define a `webserver_workers` configuration as the end result will be 2 charm config with `webserver_workers` and in such case I expect charmcraft to raise the error.
So this won't reach the charm code, therefore this case will never happen